src/components/modals/createTaskModal.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { useState, useEffect } from "react";
```

Comment:
This is the create task modal which at the moment is fully functional depending on user selecting the start time and start date and also end time and end date.
This is an issue from the backend since it has not been configured to also accept dates without the time included.